<ui version="4.0" >
 <class>Isibus</class>
 <widget class="QMainWindow" name="Isibus" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="1" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_etat_reseau" >
       <attribute name="title" >
        <string>Etat du réseau</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QTableWidget" name="tableWidget" >
          <column>
           <property name="text" >
            <string>Ligne</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Bus</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Etat</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_property" >
       <attribute name="title" >
        <string>Propriétés du bus</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="2" column="0" colspan="2" >
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title" >
           <string>Piloter le bus</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="0" column="0" >
            <widget class="QPushButton" name="slowBusButton" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Faire &amp;ralentir ce bus</string>
             </property>
             <property name="icon" >
              <iconset>../../build/ui/pics/bus_Slow.png</iconset>
             </property>
             <property name="iconSize" >
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QPushButton" name="accelBusButton" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Faire &amp;accelerer ce bus</string>
             </property>
             <property name="icon" >
              <iconset>../../build/ui/pics/bus_Accel.png</iconset>
             </property>
             <property name="iconSize" >
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0" colspan="2" >
         <widget class="QPushButton" name="delBusButton" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>&amp;Supprimer ce bus</string>
          </property>
          <property name="icon" >
           <iconset>../../build/ui/pics/bus_delete.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" >
         <widget class="QPushButton" name="button_viewbus" >
          <property name="text" >
           <string>Afficher ce bus</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QComboBox" name="cbBus" />
        </item>
        <item row="1" column="0" colspan="2" >
         <widget class="QGroupBox" name="groupBox_3" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>5</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title" >
           <string>Information sur le bus selectionné</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="4" column="0" colspan="2" >
            <widget class="QGroupBox" name="groupBox_4" >
             <property name="title" >
              <string>Route / Segment</string>
             </property>
             <layout class="QGridLayout" >
              <property name="margin" >
               <number>9</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item row="0" column="1" >
               <widget class="QLCDNumber" name="lcdn_segment" />
              </item>
              <item row="0" column="0" >
               <widget class="QLCDNumber" name="lcdn_route" />
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLCDNumber" name="lcdn_numero" />
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>Numéro du bus</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLCDNumber" name="lcdn_ligne" />
           </item>
           <item row="2" column="1" >
            <widget class="QLCDNumber" name="lcdn_capacite" />
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Ligne</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QLCDNumber" name="lcdn_vitesse" />
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Capacité</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="label_4" >
             <property name="text" >
              <string>Vitesse (km/h)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_arret" >
       <attribute name="title" >
        <string>Arrêt</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="2" colspan="2" >
         <widget class="QPushButton" name="button_viewarret" >
          <property name="text" >
           <string>Afficher l'arret</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2" >
         <widget class="QComboBox" name="cb_arret" />
        </item>
        <item row="1" column="0" >
         <widget class="QGroupBox" name="gbBus1" >
          <property name="title" >
           <string>Bus Arrivant 1</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="0" >
            <widget class="QLabel" name="lcdArretBusStatus1" >
             <property name="text" >
              <string>Status</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QLCDNumber" name="lcdArretBus1" />
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="4" >
         <widget class="QGroupBox" name="groupBox" >
          <property name="title" >
           <string>Pasager</string>
          </property>
          <widget class="QPushButton" name="pushButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="geometry" >
            <rect>
             <x>11</x>
             <y>93</y>
             <width>451</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text" >
            <string>Ajouter un passager à cet arret</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="1" colspan="2" >
         <widget class="QGroupBox" name="gbBus2" >
          <property name="title" >
           <string>Bus Arrivant 2</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="0" >
            <widget class="QLabel" name="lcdArretBusStatus2" >
             <property name="text" >
              <string>Status</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QLCDNumber" name="lcdArretBus2" />
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QGroupBox" name="gbBus3" >
          <property name="title" >
           <string>Bus Arrivant 3</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="1" column="0" >
            <widget class="QLabel" name="lcdArretBusStatus3" >
             <property name="text" >
              <string>Status</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QLCDNumber" name="lcdArretBus3" />
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_historique" >
       <attribute name="title" >
        <string>Historique de la session</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QListWidget" name="lw_historique" />
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QPushButton" name="addBusButton" >
      <property name="text" >
       <string>Ajouter un &amp;Bus...</string>
      </property>
      <property name="icon" >
       <iconset>pics/bus_new.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item rowspan="2" row="0" column="0" >
     <widget class="QGraphicsView" name="mainView" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>&amp;A propos</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_Quit" />
   </widget>
   <widget class="QMenu" name="menu_Gestion_d_v_nements" >
    <property name="title" >
     <string>&amp;Gestion d'évènements</string>
    </property>
    <addaction name="actionProvoquer_la_panne_d_un_bus" />
    <addaction name="actionProvoquer_une_m_eute_dans_un_bus" />
    <addaction name="actionCr_er_un_bouchon_sur_le_r_seau_routier" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Gestion_d_v_nements" />
   <addaction name="menuAbout" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Quit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionProvoquer_la_panne_d_un_bus" >
   <property name="text" >
    <string>Provoquer la &amp;panne d'un bus</string>
   </property>
  </action>
  <action name="actionProvoquer_une_m_eute_dans_un_bus" >
   <property name="text" >
    <string>Provoquer une ém&amp;eute dans un bus</string>
   </property>
  </action>
  <action name="actionCr_er_un_bouchon_sur_le_r_seau_routier" >
   <property name="text" >
    <string>Créer un &amp;bouchon sur le réseau routier</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
